cypress/support/config/services.js

```diff
@@ -873,6 +873,30 @@ module.exports = {
       mediaAssetPage: { path: undefined, smoke: false },
     },
   },
+  scotland: {
+    font: undefined,
+    isWorldService: false,
+    variant: 'default',
+    pageTypes: {
+      articles: {
+        path:
+          Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
+            ? undefined
+            : '/scotland/articles/czwj5l0n210o',
+        smoke: false,
```

Comment:
Since this is a page that has different configuration than the others - in terms of ATI analytics and styling, it'd be best to add this to the smoke test suite that runs on each PR.
```suggestion
        smoke: true,
```